service/modules/logreg-module/conda.yaml

```diff
@@ -0,0 +1,10 @@
+name: tutorial
+channels:
+  - defaults
+dependencies:
+  - pip:
+    - mlflow
+    - pandasql
```

Comment:
I don't know if `pandasql` is necessary here? I don't think you're making any sql queries so I'm not sure. Something worth looking into.

Reply:
It is not. We use it for the sql module

Reply:
Thanks for catching it!